src/v/cluster/tests/partition_balancer_simulator_test.cc

```diff
@@ -961,7 +967,7 @@ FIXTURE_TEST(test_mixed_replication_factors, partition_balancer_sim_fixture) {
 
     add_node(model::node_id{4}, 300_GiB, 1);
     add_node_to_rebalance(model::node_id{4});
-    BOOST_REQUIRE(run_to_completion(1000));
+    BOOST_REQUIRE(run_to_completion(total_replicas() * 3));
```

Comment:
I think we can make this the default everywhere and get rid of the `max_balancer_actions` parameter completely.

Reply:
i will do it in the follow up PR, to make the test stable